dev-manual/api/_code/start_transfer.curl

```diff
@@ -0,0 +1,16 @@
+curl --request POST \
+  --url http://am110rpm.qa.archivematica.net:81/api/transfer/start_transfer/ \
+  --header 'Accept: */*' \
+  --header 'Authorization: ApiKey test:eb9b3934c56de5c4eb22b2649637c9416e8912f1' \
+  --header 'Cache-Control: no-cache' \
+  --header 'Connection: keep-alive' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --header 'Host: am110rpm.qa.archivematica.net:81' \
+  --header 'Postman-Token: 905b608d-c954-4a1f-85db-f5fd29350072,273ac01a-b726-46cd-85f7-cac4c85db140' \
+  --header 'User-Agent: PostmanRuntime/7.15.0' \
+  --header 'accept-encoding: gzip, deflate' \
+  --header 'cache-control: no-cache' \
+  --header 'content-length: 228' \
+  --header 'cookie: csrftoken=vCJt5dUB9WnKKgLAFN8mEoyrFiZSGm0D' \
+  --cookie csrftoken=vCJt5dUB9WnKKgLAFN8mEoyrFiZSGm0D \
+  --data 'name=mc_api_transfer&type=standard&accession=2019-1234&paths%5B%5D='\''M2JlYzc0MmYtMjBjMy00ZjBlLThjMzEtZGFmYTVlYzE4ZmRmOmNlbnRvcy9hcmNoaXZlbWF0aWNhLXNhbXBsZWRhdGEvU2FtcGxlVHJhbnNmZXJzL0RlbW9UcmFuc2ZlckNTVg%3D%3D'\''&rows_id%5B%5D='
```

Comment:
I think we can cut this request to:
```bash
curl --request POST \
  --url http://am110rpm.qa.archivematica.net:81/api/transfer/start_transfer/ \
  --header 'Authorization: ApiKey test:eb9b3934c56de5c4eb22b2649637c9416e8912f1' \
  --data 'name=mc_api_transfer&type=standard&accession=2019-1234&paths%5B%5D='\''M2JlYzc0MmYtMjBjMy00ZjBlLThjMzEtZGFmYTVlYzE4ZmRmOmNlbnRvcy9hcmNoaXZlbWF0aWNhLXNhbXBsZWRhdGEvU2FtcGxlVHJhbnNmZXJzL0RlbW9UcmFuc2ZlckNTVg%3D%3D'\''&rows_id%5B%5D='
```

Do you think it might be useful for folks if we elaborate on how to get the path they need into b64? I know it's something I'm always having to look through notes to find again. e.g. `[$(echo -n '6b82c1f5-2b87-49ba-8e5f-947759201518:/home/archivematica/archivematica-sampledata/api-demo/' \
    | base64 -w 0)]"`

As a side note, I recognize this is still a work in progress, but as we get closer to publication, perhaps we should figure out what might be a good standard URL to call in examples, e.g. we've used sandbox and am110 in examples, but maybe we can make a mock url? (or example.com??)

I wonder too if we should mark certain attributes as optional, e.g. `row_id` if it isn't included in the request then it will still work.